OpenDreamRuntime/DreamThread.cs

```diff
@@ -479,7 +479,7 @@ private void HandleException(Exception exception) {
                 line = source.Item2;
             }
 
-            dreamMan.HandleException(exception, msg, file, line);
+            dreamMan.HandleException(exception, msg, file, line, inWorldError: _current?.Proc?.OwningType == dreamMan.WorldInstance.ObjectDefinition.TreeEntry && _current.Proc.Name == "Error");
```

Comment:
If `/world/Error()` calls another proc and _that_ one errors, it also needs to not call `/world/Error()` again.